### PR TITLE
Fixed script, docs, documented features

### PR DESCRIPTION
**Pull request by andoresuperesu**
Updated documentation, organized a little how the script works, documented some undocumented parts, fixed the command call to current non-deprecated Inkscape commands, updated the `.svg` template to use open fonts.

### CODE REVIEW

File: certg.py [hunk 1]
```diff
@@ -67,7 +67,8 @@ def main():
         # Below you need to add the route to the executable of inkscape
         # e.g Windows: C:/Program Files/Inkscape/inkscape.exe
         # e.g Mac: /Applications/Inkscape.app/Contents/MacOS/inkscape
-        cmd = ["/Applications/Inkscape.app/Contents/MacOS/inkscape", '--export-pdf={}'.format(result), tmpfile]
+        # e.g Linux: inkscape
+        cmd = ['inkscape', '--export-type=pdf', f'--export-filename={result}', tmpfile]
```
Comment (pyjavo) [inline certg.py:71]: Please don't erase the MAC path. Instead, using a built-in python module, detect the Operating System and in accordance to the answer, set a specific path. For example:

```
import os

if os.name == "nt":
    path = C:/Program Files/Inkscape/inkscape.exe
    cmd = [path, "...."]
elif os.name == "posix":
    path = cmd = ["/Applications/Inkscape.app/Contents/MacOS/inkscape", '--export-pdf={}'.format(result), tmpfile]
    # cmd = [path, "...."]
# same validation for Linux
```

Also, check if another module like "platform" identifies better the different OS names https://docs.python.org/3/library/os.html#os.name

File: Instrucciones.txt
Comment (pyjavo) [inline Instrucciones.txt:1]: This information was moved to another file? Would you mind explaining me why you deleted it?

**Review by pyjavo — CHANGES_REQUESTED** [review]

File: certg.py (hunk 1 above)
Comment (andoresuperesu) [reply certg.py:71]: Fork the original project that already includes all these. Mac wasn't deleted, it's in comments as windows and Linux is. It's an script intended for manual editing.

File: Instrucciones.txt
Comment (andoresuperesu) [reply Instrucciones.txt:1]: It's in the read me as it should be.

File: certg.py (hunk 1 above)
Comment (pyjavo) [reply certg.py:71]: Alright, let's improve it then. We can make python detect the OS and choose a path.
My goal with the proposal is to remove the manual editing :sweat_smile:

Comment (pyjavo) [reply certg.py:71]: And, for the record, you can take your time. We will use this after the event (probably in December).

Comment (andoresuperesu) [reply certg.py:71]: The initial PR was only to get it to work as is and organize it and document it all. It's done. You could add an issue or reject the PR. I'm done with this one. 😅

**Review by pyjavo — APPROVED** [review]

**Pull request merged**